### PR TITLE
First version of release notes for 0.12

### PR DESCRIPTION
**Pull request by emmanuelle**
I might have forgotten important highlights of the release -- please comment!

### CODE REVIEW

**Comment by blink1073** [comment]
:+1:, very nice!

File: doc/release/release_0.12.txt [hunk 1]
```diff
@@ -0,0 +1,151 @@
...
+- The measure of structural similarity (``measure.compare_ssim``) is now
+  n-dimensional and supports color channels as well.
+- We fixed an issue related to incorrect propagation in plateaus in
+  ``segmentation.watershed``
+
+Contributors to this release
+----------------------------
+(Listed alphabetically by last name)
+
+- Ömer Özak
```
Comment (ahojnnes) [inline doc/release/release_0.12.txt]: This would go after Juan :-)

**Comment by ahojnnes** [comment]
Other than that, looks great!

File: doc/release/release_0.12.txt [hunk 2]
```diff
@@ -0,0 +1,151 @@
...
+  ``restoration.denoise_nl_means``
+- ``measure.LineModel`` was deprecated in favor of ``measure.LineModelND``
+- ``measure.structural_similarity`` has been renamed
+  ``measure.compare_ssim``
+- ``data.lena`` has been deprecated, and gallery examples use instead the
+  ``data.astronaut()`` picture.
+- The measure of structural similarity (``measure.compare_ssim``) is now
+  n-dimensional and supports color channels as well.
+- We fixed an issue related to incorrect propagation in plateaus in
+  ``segmentation.watershed``
```
Comment (soupault) [inline doc/release/release_0.12.txt]: why is this in `deprecations`?

File: doc/release/release_0.12.txt [hunk 3]
```diff
@@ -0,0 +1,151 @@
...
+- ``filters.gabor_filter`` has been renamed ``filters.gabor``
+- ``restoration.nl_means_denoising`` has been renamed
+  ``restoration.denoise_nl_means``
+- ``measure.LineModel`` was deprecated in favor of ``measure.LineModelND``
+- ``measure.structural_similarity`` has been renamed
+  ``measure.compare_ssim``
+- ``data.lena`` has been deprecated, and gallery examples use instead the
+  ``data.astronaut()`` picture.
+- The measure of structural similarity (``measure.compare_ssim``) is now
+  n-dimensional and supports color channels as well.
```
Comment (soupault) [inline doc/release/release_0.12.txt]: Same here.

**Comment by emmanuelle** [comment]
@soupault my bad, should be corrected now

**Comment by blink1073** [comment]
Rolling.

**Comment by soupault** [comment]
#1177 should really have been mentioned :beers: :)

**Comment by emmanuelle** [comment]
@soupault don't hesitate to make another pull request for more completeness

**Pull request merged**